src/Server/middleware/hardcodedRedirects.ts

```diff
@@ -71,6 +71,8 @@ const PERMANENT_REDIRECTS = {
   "/user/payments": "/settings/payments",
   "/user/alerts": "/favorites/alerts",
   "/settings/alerts": "/favorites/alerts",
+  "/settings/alerts/:alertID/edit": "/favorites/alerts/:alertID/edit",
+  "/settings/alerts/:alertID/artworks": "/favorites/alerts/:alertID/artworks",
   "/alerts/:alertID/edit": "/favorites/alerts/:alertID/edit",
   "/alerts/:alertID/artworks": "/favorites/alerts/:alertID/artworks",
```

Comment:
I could be wrong, but I believe the `/alerts` redirects were added in #13820,  but we could keep them as well :)
```suggestion
```